src/components/Shared/SendModal/SendLN/SendLN.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { I18nextProvider } from "react-i18next";
+import i18n from "../../../../i18n/test_config";
+import SendLN from "./SendLN";
+
+const basicProps = {
+  balance: "123456",
+  onConfirm: () => {},
+  onChangeInvoice: () => {},
+};
+
+describe("SendLN", () => {
+  it("renders", async () => {
```

Comment:
Please use `test` (just another word, same functionality), to make it consistent with the rest of the app.

E.g.: https://github.com/cstenglein/raspiblitz-web/blob/master/src/util/format.spec.ts

Reply:
Done, I prefer the `it` because it reminds you to write a full sentence. But nevermind.